pkg/api/v1/testkube/model_variables_extended.go

```diff
@@ -20,3 +18,31 @@ func VariablesToMap(v Variables) map[string]string {
 
 	return vars
 }
+
+func ObfuscateSecrets(output string, variables Variables, testName string) string {
+	// TODO: this is ugly, does anybody have a better idea?
+	namespace := "testkube"
+	if ns, ok := os.LookupEnv("TESTKUBE_NAMESPACE"); ok {
+		namespace = ns
+	}
+	secretClient, err := secret.NewClient(namespace)
+	var secretKeyValues map[string]string
+	if err == nil {
+		secretKeyValues, _ = secretClient.Get(secret.GetMetadataName(testName))
```

Comment:
It works this way only for git credentials. But for secret variables we need to take the name of secret and the secret key from SecretRef object. Also, check the error, if it's something rather than not found